includes/oauth-php/OAuth.php

```diff
@@ -100,17 +100,17 @@ public function check_signature($request, $consumer, $token, $signature) {
     // Avoid a timing leak with a (hopefully) time insensitive compare
     $result = 0;
     for ($i = 0; $i < strlen($signature); $i++) {
-      $result |= ord($built{$i}) ^ ord($signature{$i});
+      $result |= ord($built{$i}) ^ ord($signature[$i]);
     }
 
     return $result == 0;
   }
 }
 
 /**
- * The HMAC-SHA1 signature method uses the HMAC-SHA1 signature algorithm as defined in [RFC2104] 
- * where the Signature Base String is the text and the key is the concatenated values (each first 
- * encoded per Parameter Encoding) of the Consumer Secret and Token Secret, separated by an '&' 
+ * The HMAC-SHA1 signature method uses the HMAC-SHA1 signature algorithm as defined in [RFC2104]
```

Comment:
Should we do these changes as a separate whitespace commit/PR?

Reply:
Sorry, my editor got a little too helpful it looks like.